sema/harvest/__main__.py

```
@@ -169,13 +156,13 @@ def init_load(args: argparse.Namespace, store: RDFStore):
     store.insert(graph, "urn:harvest:context")
 
 
-def make_service(args) -> service:
+def make_service(args) -> HarvestService:
     store_info: list = args.store or []
     log.debug(f"{args.store}")
```

Ruff

Logging statement uses f-string

(G004)